src/components/Icons/paths.js

```diff
@@ -0,0 +1,37 @@
+/** Custom Icon paths */
+export const ICONS = {
+  Heart:
+    'M20.84 4.60999C20.3292 4.099 19.7228 3.69364 19.0554 3.41708C18.3879 3.14052 17.6725 2.99817 16.95 2.99817C16.2275 2.99817 15.5121 3.14052 14.8446 3.41708C14.1772 3.69364 13.5708 4.099 13.06 4.60999L12 5.66999L10.94 4.60999C9.9083 3.5783 8.50903 2.9987 7.05 2.9987C5.59096 2.9987 4.19169 3.5783 3.16 4.60999C2.1283 5.64169 1.54871 7.04096 1.54871 8.49999C1.54871 9.95903 2.1283 11.3583 3.16 12.39L4.22 13.45L12 21.23L19.78 13.45L20.84 12.39C21.351 11.8792 21.7563 11.2728 22.0329 10.6053C22.3095 9.93789 22.4518 9.22248 22.4518 8.49999C22.4518 7.77751 22.3095 7.0621 22.0329 6.39464C21.7563 5.72718 21.351 5.12075 20.84 4.60999V4.60999Z',
+  HeartFull:
+    'M4.42144 4.04047C7.60919 1.824 10.7645 4.57252 11.6102 5.63662C13.7246 3.90746 18.7145 1.48664 21.7592 5.63662C24.8039 9.7866 16.2618 17.4747 11.6102 20.8C7.66344 15.7455 -3.61312 9.62698 4.42144 4.04047Z',
+  File:
+    'M13 2H6C5.46957 2 4.96086 2.21071 4.58579 2.58579C4.21071 2.96086 4 3.46957 4 4V20C4 20.5304 4.21071 21.0391 4.58579 21.4142C4.96086 21.7893 5.46957 22 6 22H18C18.5304 22 19.0391 21.7893 19.4142 21.4142C19.7893 21.0391 20 20.5304 20 20V9L13 2Z',
+  Youtube:
+    'M22.54 6.42C22.4212 5.94541 22.1793 5.51057 21.8387 5.15941C21.498 4.80824 21.0708 4.55318 20.6 4.42C18.88 4 12 4 12 4C12 4 5.12002 4 3.40002 4.46C2.92927 4.59318 2.502 4.84824 2.16137 5.19941C1.82074 5.55057 1.57881 5.98541 1.46002 6.46C1.14524 8.20556 0.991258 9.97631 1.00002 11.75C0.988802 13.537 1.14279 15.3213 1.46002 17.08C1.59098 17.5398 1.83833 17.9581 2.17817 18.2945C2.518 18.6308 2.93884 18.8738 3.40002 19C5.12002 19.46 12 19.46 12 19.46C12 19.46 18.88 19.46 20.6 19C21.0708 18.8668 21.498 18.6118 21.8387 18.2606C22.1793 17.9094 22.4212 17.4746 22.54 17C22.8524 15.2676 23.0063 13.5103 23 11.75C23.0112 9.96295 22.8573 8.1787 22.54 6.42V6.42Z M9.75 15.02L15.5 11.75L9.75 8.48001V15.02Z',
```

Comment:
Make these individual exports or have ICONS as the defualt export.

Reply:
**"Icons" component refactored. './Icons/index.js' serve all the exports, for ex:**
`export { default as File } from './File';`
`export { default as Heart } from './Heart';`
`export { default as HeartFull } from './HeartFull';`
**Import is now straightforward, for ex:**
`import { File, Heart, ArrowCircleLeft, HeartFull, Share } from '../Icons';`
**And we can simply call required Icon**
`<File />`

**"icon.styles.js" is used to hold style for SVG tag for all icons to avoid code repetition.**
**There is an error with 'Label' icon. While rendered** `<Label />` **i am reciving an error inside Terminal reguesting 'href' attribute , hence Icon is renamed to 'Labels',** `<Labels />`